src/PropertyInfo/Extractor/MappingReflectionExtractor.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace OpenEuropa\EPoetry\PropertyInfo\Extractor;
+
+use OpenEuropa\EPoetry\Type\ContactPersonIn;
+use OpenEuropa\EPoetry\Type\Contacts;
+use OpenEuropa\EPoetry\Type\CreateRequests;
+use OpenEuropa\EPoetry\Type\LinguisticRequestIn;
+use Symfony\Component\PropertyInfo\Extractor\ReflectionExtractor;
+use Symfony\Component\PropertyInfo\Type;
+
+/**
+ * Class MappingReflectionExtractor.
+ */
+class MappingReflectionExtractor extends ReflectionExtractor
+{
+    /**
+     * The mapping list.
```

Comment:
We should add instructions on how to add new entries in this property, ideally contributors should be able to understand how to extend this by simply reading the description.

Reply:
Fixed.